lib/datapacktypes/vlocityuitemplate.js

```diff
@@ -0,0 +1,26 @@
+var async = require('async');
+
+var VlocityUITemplate = module.exports = function(vlocity) {
+	this.vlocity = vlocity;
+};
+
+VlocityUITemplate.prototype.onDeployFinish = async function(jobInfo) {
+    if (!this.vlocity.sfdxUsername) {
+        VlocityUtils.error('Deploy Error', 'Please run again using SFDX command');
```

Comment:
Can you make this message more clear? It should not be an error. Use VlocityUtils.report

VlocityUtils.report('Invalid Action', 'OmniScripts cannot be automatically activated unless using SFDX Authentication');